### ISSUE TITLE
Hackathon Screen Shot Feedback

### ISSUE THREAD

**Issue by dawndelatte** [crowjm/tiy_assignments#12]
I think this design is really great. You can tell you took a lot of time on it, especially with adding those carrots. The colors and fonts work well with the simple and friendly vibe. The only thing I might suggest would be lightening up that dark green color on the headers, because it's a little hard to distinguish from black. Also, you might consider consolidating the recommendations and journal nav buttons into two drop down buttons in the future to simplify things for the user.

**Comment by jesse-stew** [comment]
I totally agree with the dark green header. Could you elaborate more on what you mean by consolidating those two buttons into drop down buttons? I'm having a hard time visualizing.

**Comment by dawndelatte** [comment]
So I just forked a drop down menu I was looking at on codepen [here](http://codepen.io/dawndelatte/pen/YPjyjN?editors=110). It may not be necessary, just an idea to keep the nav bar a more simple.
